docs/content/concepts/assets/graph-backed-assets.mdx

```diff
@@ -5,29 +5,29 @@ description: Defining a software-defined asset with multiple discrete computatio
-
-Graph-backed assets are useful if you have an existing graph that produces and consumes assets. Wrapping your graph inside a software-defined asset gives you all the benefits of software-defined assets — like cross-job lineage — without requiring you to change the code inside your graph.
-
-Additionally, graph-backed assets allow you to reuse an existing op across other graph-backed assets and jobs, or within the same graph.
+[Basic software-defined assets](/concepts/assets/software-defined-assets#a-basic-software-defined-asset) are computed using a single op. If generating an asset involves multiple discrete computations, you can use graph-backed assets by separating each computation into an op and building a graph to combine your computations. This allows you to launch re-executions of runs at the op boundaries, but doesn't require you to link each intermediate value to an asset in persistent storage.
 
 ---
 
 ## Relevant APIs
 
-| Name                                                       | Description                                   |
-| ---------------------------------------------------------- | --------------------------------------------- |
-| <PyObject object="AssetsDefinition" method="from_graph" /> | Constructs an asset given a graph definition. |
+| Name                                                       | Description                                                                                                                                                              |
+| ---------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| <PyObject object="graph_asset" decorator />                | Decorator for defining an asset that's computed using a graph of ops. The dependencies between the ops are specified inside the body of the decorated function.          |
+| <PyObject object="graph_multi_asset" decorator />          | Decorator for defining a set of assets that are computed using a graph of ops. The dependencies between the ops are specified inside the body of the decorated function. |
+| <PyObject object="AssetsDefinition" method="from_graph" /> | Constructs an asset, given a graph definition. Useful if you have a single graph that you want to use to power multiple different assets.                                |
 
 ---
 
 ## Defining graph-backed assets
 
-To define a graph-backed asset, use the `from_graph` attribute on the <PyObject object="AssetsDefinition" /> object. The value returned from the graph that becomes the graph-backed asset will be stored in persistent storage as the asset:
+To define a graph-backed asset, use the <PyObject object="graph_backed_asset" decorator /> decorator. The decorated function defines the dependencies between a set of ops, which are combined to compute the asset.
```

Comment:
Is it `graph_backed_asset` as it is here, or `graph_asset` as in the examples and the relevant APIs table?

Reply:
Yikes - thanks for catching